### PR TITLE
Feature/cards use grid

### PR DESCRIPTION
**Pull request by Akswii**
Removed some useless card styling and made the component more generic. Updated documentation to include use with our grid system.

### CODE REVIEW

File: src/App/Documentation/components/Card/index.js [hunk 1]
```diff
@@ -7,29 +7,25 @@ import MediaObject from "@/MediaObject";
...
+        <h2 id="overview">Overview</h2>
         <p>
-            Wrap indiviual cards in the <Property value=".cards" /> class to display them lined up next to each other.
-            When the window reaches <Link to="/docs/core/breakpoints">breakpoint</Link> sm the cards will switch to mobile view and take up one row each and stretch to fill the width of the container.
-            To decide how your cards will look use <Property value=".card-default" />, <Property value=".card-brand" />, or <Property value=".card-light" />
+            We offer three different card variants; <Property value=".card-primary" />, <Property value=".card-secondary" /> and <Property value=".card-plain" />.
         </p>
         <ComponentPreview language="html" showCasePanel codeFigure>
-            <div className="cards">
+            <div className="row">
```
Comment (hmosvoll) [inline src/App/Documentation/components/Card/index.js]: Do we need this row?

File: src/App/components/Card/index.js [hunk 2]
```diff
@@ -11,20 +11,14 @@ const Card = ({ type, title, imgSrc, text, content, textSection, smallText, btnT
...
             </header>
             : null}
-        <div className="card-body">
-            {children ?
-                <div className="highlight">
-                    {children}
-                </div>
-                : null
-            }
+        <div className={`card-body ${bodyClass ? ` ${bodyClass}` : ""}`}>
```
Comment (hmosvoll) [inline src/App/components/Card/index.js]: Seems to be an unnecessary space here.

File: src/App/components/Card/index.js [hunk 3]
```diff
@@ -11,20 +11,14 @@ const Card = ({ type, title, imgSrc, text, content, textSection, smallText, btnT
                 {title}
             </header>
```
Comment (hmosvoll) [inline src/App/components/Card/index.js]: What was our conclusion on panel header class? Maybe this should follow the same solution?

Comment (Akswii) [reply src/App/components/Card/index.js]: Remove only using <headers> and <footers>, use only classes. Sure i just forgot 😅

File: src/App/Documentation/components/Card/index.js (hunk 1 above)
Comment (Akswii) [reply src/App/Documentation/components/Card/index.js]: Nope, must have missed it when removing the grid.

**Comment by Akswii** [comment]
> 
> 
> Well, the cards were quite wide, not to sure what to do though 🤔

Not quite sure what you're responding to here 🤔

**Pull request merged**